app/controllers/v0/profile/service_histories_controller.rb

```diff
@@ -49,6 +50,14 @@ def raise_error!
           source: self.class.to_s
         )
       end
+
+      def report_results(response)
+        if response.present?
+          StatsD.increment("#{EMIS::Service::STATSD_KEY_PREFIX}.has_service_history", tags: ['true'])
```

Comment:
The tags are key:value and have to be sent as strings in a list. So here it would be `tags: ['has_service_history:true']` or something like it.